### PR TITLE
Keep the app narrow on desktop

### PR DESCRIPTION
**Pull request by stephanebisson**
https://phabricator.wikimedia.org/T295122

https://wikimedia.github.io/wikistories-prototype/T295122/#/

### CODE REVIEW

**Comment by medied** [comment]
Looks like the loading bar is not bouncing from side to side like it does in main

**Comment by jpita** [comment]
> Looks like the loading bar is not bouncing from side to side like it does in main

works fine on my machine 😂

**Comment by jpita** [comment]
nevermind, ignore my last comment, was thinking about the "time countdown bar" while showing the story

File: src/App.vue [hunk 1]
```diff
@@ -28,14 +28,16 @@ body {
   background-color: black;
   margin: 0;
   padding: 0;
+  text-align: center;
```
Comment (hueitan) [inline src/App.vue:31]: seems like this is not necessary, can someone confirm?

**Review by hueitan — APPROVED** [review]
Cool

**Pull request merged**